.storybook/themes/news.scss

```
@@ -812,6 +812,53 @@
 					list-style-position: inside,
 					list-style-type: disc,
 				),
+				author-bio: (
+					gap: var(--global-spacing-4),
+				),
+				author-bio-author: (
+					gap: var(--global-spacing-4),
+					display: flex,
+					flex-direction: row,
+					components: (
+						image: (
+							max-width: var(--global-spacing-12),
+							max-height: var(--global-spacing-12),
+							border-radius: var(--global-border-radius-2),
+						),
+						stack: (
+							flex: 1,
+						),
+					),
+				),
+				author-bio-social-link-wrapper: (
+					display: flex,
+					flex-wrap: wrap,
+					components: (
+						link: (
+							margin: var(--global-spacing-2) var(--global-spacing-4) 0 0,
+						),
+					),
+				),
+				author-bio-social-link-hover: (
+					components: (
+						icon: (
+							fill: var(--color-primary-hover),
+						),
+					),
+				),
+				author-bio-author-name: (
+					font-family: var(--font-family-primary),
+					font-size: var(--global-font-size-6),
+					font-weight: var(--heading-level-3-font-weight),
+					margin-bottom: var(--global-spacing-2),
+					line-height: var(--heading-level-6-line-height),
```

Figma seems messed up here... I believe this should be `headling-level-6`
```suggestion
					font-family: var(--font-family-primary),
					font-size: var(--heading-level-6-font-size),
					font-weight: var(--heading-level-6-font-weight),
					line-height: var(--heading-level-6-line-height),
					margin-bottom: var(--global-spacing-2),
```

---

Even in news.scss it's like `heading-level-6-line-height`

---

Updated this based on heading-level